src/main/resources/migrations/00043_create_table_oauth_states.sql

```diff
@@ -0,0 +1,15 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_table-oauth_states
+CREATE TABLE oauth_states (
+  id BIGSERIAL PRIMARY KEY,
+  gateway_account_external_id VARCHAR(26) NOT NULL,
```

Comment:
If we refer to another table within the same database, we generally use the database ID (usually the `id` column), not the external ID (which we use if we’re referring to an entity in another database)

Reply:
Changed it to reference `id` in `gateway_account`, thanks for suggesting it